src/BW/Vkontakte.php

```diff
@@ -80,6 +80,11 @@ class Vkontakte
     private static $connection;
 
     /**
+     * @var bool
+     */
+    private $isIPv6Disabled = false;
```

Comment:
Let's add `isIPv6Disabled()` public method for this private property. Otherwise we can't get access to it outside this class